python/sdk/prelude_sdk/controllers/build_controller.py

```diff
@@ -94,6 +92,19 @@ def upload(self, test_id, filename, data):
             return res.json()
         raise Exception(res.text)
 
+    @verify_credentials
+    def compile_code_string(self, code: str):
+        """ Compile a code string """
+        res = self._session.post(
+            f'{self.account.hq}/build/compile',
+            json=dict(code=code),
+            headers=self.account.headers,
+            timeout=10
+        )
+        if res.status_code == 200:
+            return res.json()
+        raise Exception(res.text)
```

Comment:
is there reason to add things like this to the public sdk that we don't add to the cli? This doesn't provide much use to anyone outside of the UI- if writing a test, build it locally and then upload to a prelude test and you'll get the compile output there.

Reply:
yeah idk. it's nice to have an official e2e test of it. and maybe someone exists out there who writes go code but can't / doesnt want to compile locally... lol? pretty weak arguments though, and happy to remove if you think i should

Reply:
the testing argument has me sold. Carry on!